.travis.yml

```diff
@@ -11,49 +12,150 @@ script:
   - 'bundle exec rake $CHECK'
 bundler_args: --without system_tests
 rvm:
-  - 2.5.3
-stages:
-  - static
-  - spec
-  - acceptance
-  -
-    if: tag =~ ^v\d
-    name: deploy
+  - 2.5.1
```

Comment:
The change in rvm version means we are downgrading the version of ruby we are testing. 
It seems from the old config:
Puppet 5 = Ruby 2.4.5
Puppet 6 = Ruby 2.5.3